### PR TITLE
Pull request for Project 2

### PR DESCRIPTION
**Pull request by Nelliuslinda**
# Description

Created the required forms for the project.

Fixes # (issue)
Created the web forms for the project

# Type of change
Create the necessary forms using the appropriate form tags

- [x] New feature (non-breaking change which adds functionality)

## Change Details

- Feature. Created two separate form
- Both forms work okay

## Screenshots (if applicable)
image
image

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: Form_2/form.html [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+            </li>
+
+            <!--Button-->
+            <li class="button">
+                <button type="submit">Send your message</button>
+            </li>
+        </ul>
+    </form>
+</body>
+</html>
```
Comment (gertie-sheshe) [inline Form_2/form.html]: Add a new line at the end of the file :) [Here is the reason ](https://gist.github.com/camh-/1bebfcff1b0f814e9b191edc60d5206b)why this should be

File: Form_1/web_form.html [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+        <!--Third Section -->
+        <section>
+            <p>
+              <button type="submit">Validate the payment</button>
+            </p>
+        </section>
+        
+    </form>
+</body>
+</html>
```
Comment (gertie-sheshe) [inline Form_1/web_form.html]: Add a new line at the end of the file :) [Here is the reason ](https://gist.github.com/camh-/1bebfcff1b0f814e9b191edc60d5206b)why this should be

**Review by gertie-sheshe — COMMENTED** [review]
Great work! @Nelliuslinda Added a comment on adding new line at the end of the file. Other than that - all looks good 🔥

File: Form_2/form.html (hunk 1 above)
Comment (Nelliuslinda) [reply Form_2/form.html]: Thank you for the correction😊

Comment (Nelliuslinda) [reply Form_2/form.html]: I've rectified the issue

**Pull request merged**